### DISCUSSION TITLE
drifter data

### DISCUSSION THREAD

**Discussion by hroarty** [OpenDrift/opendrift#1054] [Q&A]
Any suggestions on how to extract latitude and longitude of the seeds at each time step of the model run?

**Comment by AndresSepulveda** [comment]
Hi, I am not sure what is what you want do to. You can define the integration time step and the output time step for your particles, and the latitude and longitude are stored for the particles (seeds) by default. Please clarify...

**Reply by hroarty** [reply]
thanks for responding.  How do I access the lat and lon of the particles?  I don't see the command to accomplish this?

**Comment by AndresSepulveda** [comment]
Ah, you can save them in a NetCDF output file with the option

o.run(time_step=900, time_step_output=1800, outfile=outfile)

now if you mean during the run, as part of the output log, that I don't know, sorry

**Comment by knutfrode** [comment]
During the run (within the update method of your module), positions are accessed as `self.elements.lon` and `self.elements.lat`

After the run, positions are accessed with
```
lons = self.history['lon']
lats = self.history['lat']
```
or 
`lons, lats = self.get_lonlats()`
The latter is recommended, as invalid positions (unseeded or deactivated) are then also masked.
